kotlin-generator/src/android-client.ts

```diff
@@ -25,7 +27,7 @@ inline fun <reified T> Gson.fromJson(json: JsonElement) =
     fromJson<T>(json, object : TypeToken<T>() {}.type)
 
 @Suppress("DeferredIsResult", "unused")
-class ApiClient(
+open class ApiClient(
```

Comment:
Why?

Reply:
To give inheritance ability to this class; Kotlin doesn't allow inheritance of classes that are not open

Reply:
But why would someone inherit from ApiClient?

Reply:
To change calling syntax, add properties and functions to the client entry point etc
I actually don't see why we shouldn't allow this 🤔

Reply:
I also don't see why we should allow this

Reply:
Kotlin has extension methods in case it is required. I don't think it should be open as it is not meant to be inherited (I fail to see an usecase that doen't have some other better way).

Reply:
If for some reason an app consumes two Sdkgen APIs, the imports would be awful; If this class is open, the dev could use inheritance to separate the ApiClients.